TrafficCapture/dockerSolution/src/main/docker/docker-compose.yml

```diff
@@ -35,6 +35,7 @@ services:
       - "3000:3000"
     volumes:
       - ./grafana_data:/var/lib/grafana
+      - ./grafana_datasources.yaml:/usr/share/grafana/conf/provisioning/datasources/datasources.yaml
```

Comment:
thank you for figuring this out!
Should this be a mounted file or a configuration file that's within the image?  I can't think of how I would want to change this, and it seems like it might be useful for a quick-and-dirty deployment elsewhere w/out compose.

Reply:
Yep, I think you're right. This was helpful for testing, but a user is quite unlikely to need that.